src/neural/loader.h

```diff
@@ -28,12 +28,18 @@ namespace lczero {
 using FloatVector = std::vector<float>;
 using FloatVectors = std::vector<FloatVector>;
 
+// Read from protobuf.
+FloatVectors LoadFloatsFromPbFile(const std::string& buffer);
+
 // Read space separated file of floats and return it as a vector of vectors.
-FloatVectors LoadFloatsFromFile(const std::string& filename);
+FloatVectors LoadFloatsFromFile(std::string* buffer);
 
 // Read v2 weights file and fill the weights structure.
 Weights LoadWeightsFromFile(const std::string& filename);
 
+// Read gzip compressed data into buffer.
+std::string DecompressGzip(const std::string& filename);
```

Comment:
I think this function (as well as `LoadFloats*` functions) are not really used outside of loader.

In that case I suggest removing them from .h file and putting into anonymous namespace in .cc file.